CMakeLists.txt

```diff
@@ -26,3 +28,21 @@ endif()
 include(CTest)
 
 add_subdirectory(k4ProjectTemplate)
+
+
+# Create the package config file
+include(CMakePackageConfigHelpers)
+configure_package_config_file(
+  cmake/Project_Config_Template.cmake.in
+  ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Config.cmake
+  INSTALL_DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}
+  # Adapt if other variables are needed, also modify Project_Config_template.cmake.in
+  PATH_VARS CMAKE_INSTALL_INCLUDEDIR CMAKE_INSTALL_LIBDIR)
+write_basic_package_version_file(
+  ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}ConfigVersion.cmake
+  VERSION ${PACKAGE_VERSION_MAJOR}.${PACKAGE_VERSION_MINOR}.${PACKAGE_VERSION_PATCH}
+  COMPATIBILITY SameMajorVersion )
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Config.cmake
+              ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}ConfigVersion.cmake
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${CMAKE_PROJECT_NAME} )
+# END create package config file
```

Comment:
Should we consider moving this into a separate file in the `cmake` subfolder, like we do in other packages, e.g.:
https://github.com/key4hep/EDM4hep/blob/master/CMakeLists.txt#L97-L98

Reply:
Yes, can do that

Reply:
Moved to included file